### ISSUE TITLE
Time-based conditions

### ISSUE THREAD

**Issue by crbrox** [telefonicaid/perseo-fe#205]
From [DM-1161](https://jirapdi.tid.es/browse/DM-1161)

```
when the attribute value is a timestamp in ISO8601 format (2015-12-23T02:34:52Z)
so I can check if value is before, after or equal to another given timestamp
```

**Comment by crbrox** [comment]
when the attribute value is a timestamp in ISO8601 format (2015-12-23T02:34:52Z)
so I can check if value is before, after or equal to another given timestamp

**Comment by fgalan** [comment]
Implemented by PR #208
